### PR TITLE
[16.0][IMP] use right method to access action / remove an obsolete Warning

### PR DESCRIPTION
**Pull request by nayatec**

### CODE REVIEW

File: pos_payment_change/tests/test_module.py [hunk 1]
```diff
@@ -23,6 +23,22 @@ def setUp(self):
         ]
         self.product = self.env.ref("product.product_product_3")
         self.pos_config = self.env.ref("point_of_sale.pos_config_main").copy()
+        self.pos_user = self.env["res.users"].create(
```
Comment (legalsylvain) [inline pos_payment_change/tests/test_module.py]: we could use default current "admin" user, but just remove the "Administrator / Settings" group in that test. So it is more concise, don't you think ? 

Or directly use "base.user_demo". 

self.env = self.env.sudo(self.env.ref("base.user_demo")

**Review by legalsylvain — COMMENTED** [review]
Hi olivier. 

A remark inline. Otherwise, LGTM. 
Thanks a lot !

File: pos_payment_change/tests/test_module.py (hunk 1 above)
Comment (nayatec) [reply pos_payment_change/tests/test_module.py]: will do !

**Review by legalsylvain — APPROVED** [review]
Thanks for this fix ! 

/ocabot merge patch

**Pull request merged**